test/D2L.Security.OAuth2.WebApi.IntegrationTests/Authorization/NoImpersonationAttributeTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using D2L.Services;
+using NUnit.Framework;
+
+namespace D2L.Security.OAuth2.Authorization {
+	[TestFixture]
+	internal sealed class NoImpersonationAttributeTests {

```

Comment:
Minor: want a test for no auth token? Weird case, but the `NoImpersonationAttribute` should return true.